packages/app/src/runner/event-manager.ts

```diff
@@ -235,6 +235,10 @@ export class EventManager {
       this.saveState(state)
     })
 
+    this.reporterBus.on('debug:dismiss', () => {
```

Comment:
I wonder if this might be confusing since we also have `cy.debug()`: https://docs.cypress.io/api/commands/debug

Reply:
Made it more explicit in [9074f63](https://github.com/cypress-io/cypress/pull/25265/commits/9074f631a26c33d0f83b298f7ed938dfa8653e3e)